cms/djangoapps/models/settings/course_details.py

```diff
@@ -161,7 +165,8 @@ def update_from_json(cls, course_key, jsondict, user):
         # NOTE: below auto writes to the db w/o verifying that any of the fields actually changed
         # to make faster, could compare against db or could have client send over a list of which fields changed.
         for attribute in ABOUT_ATTRIBUTES:
-            cls.update_about_item(course_key, attribute, jsondict[attribute], descriptor, user)
+            if attribute in jsondict:
```

Comment:
Although - by eye - this looks ok, because this change is outside of a feature flag (and would affect all instances, including edx.org), a quick sign off from T&L might be a good idea: @cahrens @explorerleslie